### ISSUE TITLE
Slingshot Linux Distribution | SANS Insitute

### ISSUE THREAD

**Issue by LaloQuera710** [swiftlang/swift#78346]
https://www.sans.org/tools/slingshot/

**Comment by finagolfin** [comment]
[Swift 5.8 just made it into the upcoming 25.04 packages](https://packages.ubuntu.com/source/plucky/swiftlang), does that work for you? @melizasw is working on getting newer versions in too.

**Comment by melizasw** [comment]
Yep, 5.9.2 is fighting me a bit in that it is unable to build itself, but once that is figured out it should migrate automatically to Plucky and then on to 5.10.1.

**Comment by finagolfin** [comment]
OK, glad to hear it. 😃 I took a look at your latest 5.9 commit and noticed [you backported a patch of mine](https://salsa.debian.org/pkg-llvm-team/swiftlang/-/commit/aed174aee3fe0c5d7697484004e103effa57b3f5#e724cd2dc4123120d26d8425db93618b25b9489c). However, that is an older version that has a small bug, please use the final version that made it into trunk instead, b7ec9c840. Also, [this swiftrt patch is clearly a hack](https://salsa.debian.org/pkg-llvm-team/swiftlang/-/commit/aed174aee3fe0c5d7697484004e103effa57b3f5#4aadf0045229b876456e44a0ca5253375ca397e9), let us know what problem it fixed and we should come up with something better.

Other than those two nits, thanks for all your work on this package. I don't use those Debian-based distros, but I'm sure your work will have a great effect on Swift usage in those popular distros.

**Comment by melizasw** [comment]
I'm not making use of the swiftrt patch. When I run into a build error I tend to throw everything I can find at the problem until it builds then roll back to find the minimal changes needed. That patch was never used in a released package as far as I can recall. I've just deleted it from the 5.9.2 branch that I'm working from. I also updated the other patch, thanks for the alert on that as I know it is needed. Maybe that is what is causing the error with 5.9.2 being unable to build itself? Some nonsense about not being able to find the CxxStdlibShim module.

**Comment by finagolfin** [comment]
> Maybe that is what is causing the error with 5.9.2 being unable to build itself? Some nonsense about not being able to find the CxxStdlibShim module.

Could be, that patch does affect looking for those module maps.
